kimchi/src/prover.rs

```diff
@@ -692,8 +693,7 @@ where
             };
             // permutation
             let (mut t8, bnd) = {
-                let alphas =
-                    all_alphas.get_alphas(ArgumentType::Permutation, permutation::CONSTRAINTS);
+                let alphas = all_alphas.get_alphas(ArgumentType::Permutation, CONSTRAINTS);
```

Comment:
since a lot of modules have a `CONSTRAINTS` constant it's good to keep the path `permutation::CONSTRAINTS` here

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/b2bb15aa2556a627152b1b800e7ebc5cb239a682